src/components/ProjectsCard/ProjectCard.tsx

```diff
@@ -0,0 +1,112 @@
+import {
+  CardContent, Typography, Skeleton,
+} from '@mui/material';
+import { useEffect, useState } from 'react';
+import axios from 'axios';
+import {
+  WrappBtnDone,
+  Wrapper2,
+  WrapperBtnUD,
+  WrapperDes,
+  WrapperPN,
+  WrappBtn,
+  Wrapper,
+} from './cards.styled';
+import { iProjects, iProjectTasks } from '../../interfaces';
+import { ErrorAlert } from '..';
+
+const ProjectsCard = () => {
+  const [userProjects, setUserProjects] = useState<iProjects[]>([]);
+  const [projectTasks, setProjectTasks] = useState<iProjectTasks[]>([]);
+  const [openError, setOpenError] = useState(false);
+  const [messageError, setMessageError] = useState('');
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    axios.get('/api/projects')
+      .then((res) => {
+        setUserProjects(res.data.data);
+
+        const fetchProjectTasks = res.data.data.map((project: iProjects) => axios.get(`/api/project/${project.project_id}/task`)
+          .then((task) => task.data.data));
+
+        Promise.all(fetchProjectTasks)
+          .then((tasks) => {
```

Comment:
we have learned about callback hell and how to avoid it using promise, but what about promise hell?

This [article](https://medium.com/@pyrolistical/how-to-get-out-of-promise-hell-8c20e0ab0513) will show some Promise Hell scenarios and the steps out get of it.